app/models/podcast.rb

```diff
@@ -73,6 +73,10 @@ def apple_config
     end
   end
 
+  def has_apple_feed?
+    feeds.apple.exists?
```

Comment:
So this condition will see if there's an apple feed at all, not checking for the config or publish status, just if there is a feed at all, then truncate the descriptions in the RSS feed